src/blockdev.rs

```diff
@@ -30,28 +30,36 @@ use tempfile;
 
 use crate::errors::*;
 
-pub fn mount_boot(device: &str) -> Result<Mount> {
+pub fn mount_partition_by_label(device: &str, label: &str, flags: mount::MsFlags) -> Result<Mount> {
```

Comment:
This requires the caller to know about `nix::mount`.  That seems like a reasonable tradeoff, but worth noting.